client/tracing/src/logging/mod.rs

```diff
@@ -164,8 +164,10 @@ where
 		"%Y-%m-%d %H:%M:%S%.3f".to_string()
 	});
 
-	let (telemetry_layer, telemetry_worker) =
-		sc_telemetry::TelemetryLayer::new(telemetry_buffer_size, telemetry_external_transport)?;
+	let telemetry_worker = sc_telemetry::TelemetryWorker::new(
```

Comment:
Why is that still created here? What is the reason for this?

Reply:
It can be created elsewhere. The reason is I didn't think of it :grin: Fixed in f3f7c78e79